File: rules/go/tests/go.hardcoded-secret/secret.go
package main

func secrets() {
	password := "supersecretvalue" // crit:expect go.hardcoded-secret
	apiKey := "abcdef0123456789"   // crit:expect go.hardcoded-secret
	username := "administrator"    // crit:expect-not go.hardcoded-secret
	secret := "short"              // crit:expect-not go.hardcoded-secret
	_, _, _, _ = password, apiKey, username, secret
}

File: rules/rust/tests/rs.hardcoded-secret/secret.rs
fn secrets() {
    let password = "supersecret123"; // crit:expect rs.hardcoded-secret
    let api_key = "abcdef0123456789"; // crit:expect rs.hardcoded-secret
    let username = "administrator"; // crit:expect-not rs.hardcoded-secret
    let secret = "short"; // crit:expect-not rs.hardcoded-secret
}
